SU-SWS/stanford_magazine
# Turn off "Disable SQL Rewrite" in views after core patch

# READY FOR REVIEW

# Summary
- Update all version numbers in .info files to 7.x-1.0-beta3 (WM)
- Update CHANGELOG.txt with a human-readable version of what’s been changed since 7.x-1.0-beta2 (WM)


# Needed By (04/25/2018)

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Make sure all the .info files and CHANGELOG.txt files are correct with the 7.x-1.0-beta3 update

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2850

## More Information

## Folks to notify
@minorwm
